crates/fuzzing/src/generators/config.rs

```diff
@@ -182,7 +182,7 @@ impl Config {
             .allocation_strategy(self.wasmtime.strategy.to_wasmtime())
             .generate_address_map(self.wasmtime.generate_address_map)
             .cache_call_indirects(self.wasmtime.cache_call_indirects)
-            .max_call_indirect_cache_slots(self.wasmtime.max_call_indirect_cache_slots);
+            .call_indirect_cache_size_log2(self.wasmtime.call_indirect_cache_size_log2);
```

Comment:
Might want to clamp this value to a reasonable range?

Reply:
Ah, it's already clamped in the method on `Config`, so preferred to let fuzzing maximally poke the API surface here (no special limits). I guess we could do something that avoids biasing the randomness (as-is, the `Arbitrary` range on the `u8` maps 0..=255 to 0..=16 with 15/16ths of the space going to 16) but I don't think it's too important to evenly spread effort across sizes that way...